docs/en/getting-started/step3.md

```diff
@@ -5,6 +5,21 @@ title: Importing Hosts into Qv2ray
 # Step 3: Importing Hosts into Qv2ray
 Qv2ray supports many ways of importing your configuration. To get you started, we have just picked these most common use cases.
 
+## Manual Configuration
+
+1. Click **New** in the lower left corner of the main window
+2. Fill in the **Name/Prefix** or leave it blank, or you can rename it after complete the configuration.
+3. Click **Input Manually** -> **Open Connection Editor**.
+4. Fill in the **Tag**. This tag will appear in the log. You can also leave it blank.
+5. Fill in **Host** and choose your proxy type.
+6. Config **Misc Settings** Optionally.
+7. Fill in **Outbound settings** and **Stream Settings**.
+8. Click **OK** to save your config.
+
+## Import an existing configuration
+
+If you are using v2ray-core, you can import an existing configuration file. **Advanced** -> **Select**, or **Open JSON Editor** and paste it.
+
```

Comment:
This is absolutely not correct. If you are going to import a whole JSON file, you will have to check "Import As Complex Config" to use a full JSON.

Otherwise, you will only need the outbound part of the file. Here is an example:

<details>

```json
{
    "outbounds": [
        {
            "mux": {
            },
            "protocol": "vmess",
            "sendThrough": "0.0.0.0",
            "settings": {
                "vnext": [
                    {
                        "address": "xxxxxxxxxxxxxxx",
                        "port": 1211,
                        "users": [
                            {
                                "alterId": 0,
                                "id": "xxxxxxxxxxxxxxxxxxx",
                                "level": 0,
                                "security": "auto"
                            }
                        ]
                    }
                ]
            },
            "streamSettings": {
                "dsSettings": {
                    "path": "/"
                },
                "httpSettings": {
                    "host": [
                    ],
                    "path": "/"
                },
                "kcpSettings": {
                    "congestion": false,
                    "downlinkCapacity": 20,
                    "header": {
                        "type": "none"
                    },
                    "mtu": 1350,
                    "readBufferSize": 1,
                    "seed": "",
                    "tti": 20,
                    "uplinkCapacity": 5,
                    "writeBufferSize": 1
                },
                "network": "tcp",
                "quicSettings": {
                    "header": {
                        "type": "none"
                    },
                    "key": "",
                    "security": ""
                },
                "security": "none",
                "sockopt": {
                    "mark": 0,
                    "tcpFastOpen": false,
                    "tproxy": "off"
                },
                "tcpSettings": {
                    "header": {
                        "request": {
                            "headers": {
                            },
                            "method": "GET",
                            "path": [
                            ],
                            "version": "1.1"
                        },
                        "response": {
                            "headers": {
                            },
                            "reason": "OK",
                            "status": "200",
                            "version": "1.1"
                        },
                        "type": "none"
                    }
                },
                "tlsSettings": {
                    "allowInsecure": false,
                    "allowInsecureCiphers": false,
                    "alpn": [],
                    "certificates": [],
                    "disableSessionResumption": false,
                    "disableSystemRoot": false,
                    "serverName": ""
                },
                "wsSettings": {
                    "headers": {
                    },
                    "path": "/"
                }
            },
            "tag": "outBound_PROXY"
        }
    ]
}
```

</details>

Reply:
The only outbound in the configuration file I have some thoughts on.

So far, the manual configuration of Qv2ray inbound has only supported HTTP and socks, but the user's scenarios are unpredictable and he can use the local Vmess inbound for some testing or personal use.

My personal opinion is that full configuration files are allowed in complex configurations. Use Qv2ray's preferences settings if not given in the configuration file.

Reply:
Okay. That's fine.